packages/cojson-storage/src/managerSync.ts

```diff
@@ -153,56 +144,69 @@ export class StorageManagerSync {
 
         collectNewTxs({
           newTxsInSession,
-          newContentMessages,
+          contentMessage,
           sessionRow,
           firstNewTxIdx: idx,
           signature: signature.signature,
         });
 
         idx = signature.idx + 1;
+
+        if (signatures.length > 1) {
+          await this.sendContentMessage(coValueRow, contentMessage);
+          contentMessage = {
+            action: "content",
+            id: coValueRow.id,
+            header: coValueRow.header,
+            new: {},
+            priority: cojsonInternals.getPriorityFromHeader(coValueRow.header),
+          } satisfies CojsonInternalTypes.NewContentMessage;
+
+          // Introduce a delay to not block the main thread
+          // for the entire content processing
+          await new Promise((resolve) => setTimeout(resolve));
```

Comment:
Added a syntetic delay to unblock the main thread when processing large files.

Since the `handleSyncMessage` in the sync storages isn't awaited this also means that we are going to load the rest of the chunks in background.